src/Resources/skeleton/dto/Data.tpl.php

```diff
@@ -0,0 +1,70 @@
+<?= "<?php" . PHP_EOL ?>
+<?php use Symfony\Bundle\MakerBundle\Str;
+
+?>
+
+namespace <?= $namespace ?>;
+
+<?php if (isset($bounded_full_class_name)): ?>
+use <?= $bounded_full_class_name ?>;
+<?php endif ?>
+use Symfony\Component\Validator\Constraints as Assert;
+
+/**
+ * Data transfer object for <?= $bounded_class_name ?>.
+ */
+class <?= $class_name ?>
+
+{
+<?php if ($addHelpers): ?>
+    /**
+     * Create DTO, optionally extracting data from a model.
+     */
+    public function __construct(?<?= $bounded_class_name ?> $<?= lcfirst($bounded_class_name) ?> = null)
+    {
+        if ($<?= lcfirst($bounded_class_name) ?> instanceof <?= $bounded_class_name ?>) {
+            $this->extract($<?= lcfirst($bounded_class_name) ?>);
+        }
```

Comment:
Is there a case for passing something *other* than the bound entity class? Should this be:

```
if (null !== $bound_class_name) {
```